src/fontra/views/editor/editor.js

```diff
@@ -1938,6 +1942,7 @@ export class EditorController {
     for (const overlay of document.querySelectorAll(".cleanable-overlay")) {
       overlay.classList.add("overlay-layer-hidden");
     }
+    this.setSelectedTool("hand-tool", false);
```

Comment:
Then, here we should remember the currently selected tool, so we can restore it in the space-up code.

I suggest this:
```js
      this.savedSelectedToolIdentifier = this.selectedToolIdentifier;
      this.setSelectedTool("hand-tool");
```

Reply:
Done.